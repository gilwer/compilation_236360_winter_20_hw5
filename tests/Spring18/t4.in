//arithmetic
void main() {
	printi(8);
	
	printi(100b);
	
	printi(1+2+3);
	
	printi(120b-50b-1b);
	
	printi(10-2*3);
	
	printi(10-8);
	
	printi(10/2);
	
	printi(1+400/3b);
	
}