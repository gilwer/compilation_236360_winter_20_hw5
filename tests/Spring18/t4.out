8
100
6
69
4
2
5
134
